modules/generative-openai/clients/openai.go

```diff
@@ -105,11 +108,19 @@ func (v *openai) generate(ctx context.Context, cfg moduletools.ClassConfig, prom
 		return nil, errors.Wrap(err, "generate input")
 	}
 
+	monitoring.GetMetrics().VectorizerRequests.WithLabelValues("generate", "openai").Inc()
+	startTime := time.Now()
+	defer func() {
+		monitoring.GetMetrics().VectorizerRequestDuration.WithLabelValues("generate", oaiUrl).Observe(time.Since(startTime).Seconds())
+	}()
+
 	body, err := json.Marshal(input)
 	if err != nil {
 		return nil, errors.Wrap(err, "marshal body")
 	}
 
+	monitoring.GetMetrics().VectorizerRequestSize.WithLabelValues("generate", oaiUrl).Observe(float64(len(body)))
```

Comment:
is `VectorizerRequestSize` metric only related to vectorizer modules?

Reply:
Fixed name